src/git/tag_source.go

```diff
@@ -9,6 +10,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var ErrNoReleases = errors.New("no releases found")
```

Comment:
Seems like we're introducing a new term ("releases") to this object, while we previously referred to either `Tags` or `Versions`. I think the last two make sense (`Version` is an rt concept, and `Tags` is the git concepts where `Versions` are derived). I'd avoid introducing "releases"
```suggestion
var ErrNoReleases = errors.New("no version tags found")
```

Reply:
You are right